Automattic/wp-calypso
# Guided Onboarding: Update q2 option styling

Related to https://github.com/Automattic/dotcom-forge/issues/7647

## Proposed Changes

* Adds premium badge to Guided Onboarding flow segmentation survey question 2 `Let us build your site in 4 days` option.

<img width="1030" alt="CleanShot 2024-06-11 at 00 39 25@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/bd03dbca-ecb5-4f20-9b30-3ed41b682526">



## Why are these changes being made?


* Update text to better resonate with users based on feedback: p9Jlb4-c9F-p2#comment-11699

## Testing Instructions



* Checkout branch or use live link
* Sandbox public api and apply patch: D151715-code
* Navigate to `/start/guided`
* Click Q1 `Creating a site for myself, a business, or a friend` then verify the `Let us build your site in 4 days` includes the premium badge

## Pre-merge Checklist
